src/Root.h

```diff
@@ -229,6 +231,7 @@ class cRoot
 	cMonsterConfig * m_MonsterConfig;
 
 	cCraftingRecipes * m_CraftingRecipes;
+	cRecipeMapper *    m_RecipeMapper;
```

Comment:
You are leaking this memory. For new code, prefer `std::unique_ptr<cRecipeMapper>` over raw `new` and `delete`.